rest/making-calls/template/example-1.4.x.php

```diff
@@ -0,0 +1,26 @@
+<?php
+// Get the PHP helper library from twilio.com/docs/php/install
+require_once('/path/to/twilio-php/Services/Twilio.php'); // Loads the library
+
+// Your Account Sid and Auth Token from twilio.com/user/account
+{{#accountSid}}$sid = "{{accountSid}}";{{/accountSid}}
+{{#authToken}}$token = "{{authToken}}";{{/authToken}}
```

Comment:
Most libraries (if not all) support using an (apiKey, apiSecret) to authenticate. But we can probably separate those snippets into just one that allows you to do that?